boards/mavrinux.cpp

```diff
@@ -76,7 +76,8 @@ Mavrinux::Mavrinux(mavrinux_conf_t config):
     led(),
     mavlink_serial(config.serial_udp_config),
     file_flash(),
-    config_(config)
+    config_(config),
+    state_display_mavrinux_()
```

Comment:
Would be nice to keep the config as last param, so that we can keep the last argument as default
using (parm1, ... last_param = default) in the hpp